lib/spring/test/acceptance_test.rb

```diff
@@ -83,14 +83,14 @@ def without_gem(name)
       end
 
       test "help message when called without arguments" do
-        assert_success "bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
+        assert_success "bundle exec bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
```

Comment:
I think error on travis (check [this](https://travis-ci.org/sonalkr132/spring/jobs/95848491)) can be on user side as well (?). Looking at the test won't help him a lot cause it won't be apparent that command passing in test is not actually:
```Ruby
bin/spring
```
but 
```Ruby
bundle exec bin/spring
```

Also, **some** of the commands being passed to `with_clean_env` are:
```Ruby
# gives syntax error near unexpected token `('
# can be solved by just removing '('
(gem list bundler | grep bundler) || gem install bundler
# not sure from where does this gets passed some jugglery will be needed    
bundle check || bundle update --retry=2 

## some gem install commands
## more commands from other files calling the same app.run 
```